### ISSUE TITLE
Improved channel permissions.

### ISSUE THREAD

**Issue by anishanne** [supertiger1234/Nertivia-Client#164]
Currently, you can either turn Send Messages on/off for everyone at once. (Won't effect admins.) 
We should have the option of only allowing a certain role to view/msg in a channel. And the option to hide a channel from the view of a certain role. (Admin chat)

**Comment by JDragon16** [comment]
Yeah, I think that would be a great addition since currently when it comes to every channel being visible it can have the person make a seperate server just for staff
